src/app/pages/OnDemandTvPage/index.jsx

```diff
@@ -116,6 +117,8 @@ const OnDemandTvPage = ({ pageData }) => {
   const type = 'video';
   const title = 'On-demand TV';
 
+  const placeholderSrc = getPlaceholderImageUrl(imageUrl);
```

Comment:
Do we need these variables?  These values are only used once, and they already have names in the place they're used

eg `type="video"` is clearer than `type={type}`